setup.py

```diff
@@ -52,11 +52,14 @@
       install_requires=[
           'setuptools',
           # -*- Extra requirements: -*-
+          'five.formlib',
       ],
       extras_require={
           'test': [
               'Products.CMFTestCase',
               'Products.PloneTestCase',
+              'plone.testing',
+              'plone.app.testing',
           ],
```

Comment:
Wow I didn't pay attention that it was not from there.. My bad :disappointed: .

Thanks for the quick review/merge.